CMakeLists.txt.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.2)
+
+project(googletest-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(googletest
+  GIT_REPOSITORY    https://github.com/google/googletest.git
+  GIT_TAG           master
```

Comment:
Should we be relying on master or a particular release of gtest?

Reply:
master should be fine.
That's what they recommend .  Since they follow the same strategy of having master always in release state

Reply:
> That's what they recommend.

Can you share where you found that recommendation?

It looks like they have versioned release tags and master has a lot of new commits. Why don't we use the release tag instead?
https://github.com/google/googletest/releases/tag/release-1.10.0

Reply:
What if breaking changes?

Reply:
@ahsonkhan 
https://github.com/google/googletest/blob/master/googletest/README.md#incorporating-into-an-existing-cmake-project